workshops/views.py

```diff
@@ -456,6 +457,32 @@ class OrganizationDelete(OnlyForAdminsMixin, PermissionRequiredMixin,
     success_url = reverse_lazy('all_organizations')
 
 
+#------------------------------------------------------------
+
+
+@admin_required
+def all_memberships(request):
+    '''List all memberships.'''
+    filter = MembershipFilter(
+        request.GET,
+        queryset=Membership.objects.all()
+    )
+    memberships = get_pagination_items(request, filter)
+    context = {'title' : 'All Memberships',
+               'all_memberships' : memberships,
+               'filter': filter}
+    return render(request, 'workshops/all_memberships.html', context)
+
+
+@admin_required
+def membership_details(request, membership_id):
+    '''List details of a particular membership.'''
+    membership = get_object_or_404(Membership, id=membership_id)
+    context = {'title' : '{0}'.format(membership),
+               'membership' : membership}
+    return render(request, 'workshops/membership.html', context)
```

Comment:
Should we add a link to `membership_details` view from the `organization_details` view?
Although we already have a table there, a link wouldn't hurt.